apps/web/screens/PetitionView/pagination.ts

```diff
@@ -0,0 +1,15 @@
+export const PAGE_SIZE = 3
+
+export function paginate(petitions, pageSize, pageNumber) {
+  if (petitions === undefined) return
```

Comment:
Is this done to purposefully allow petitions to pass as `null` into the else block?